stablehlo/reference/Interpreter.cpp

```diff
@@ -56,6 +56,11 @@ llvm::Expected<SmallVector<Tensor>> eval(func::FuncOp func,
       if (it != stackFrame.end()) return it->second;
       llvm::report_fatal_error("Expected a terminator when evaluating func");
     };
+    auto fetchVariadicOperands =
```

Comment:
Please rebase. With that the call to `fetchVariadicOperands` will be replaced by `scope.add`.